docs/Makefile

```diff
@@ -6,7 +6,7 @@ ifneq ($(EXAMPLES_PATTERN),)
 endif
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W -j auto $(EXAMPLES_PATTERN_OPTS)
+SPHINXOPTS    = -W -b coverage -j auto $(EXAMPLES_PATTERN_OPTS)
```

Comment:
I'm not sure this works as expected: 

python.txt is just

```Undocumented Python objects
===========================
torchvision.transforms.functional
---------------------------------
Functions:
 * scale
 ```

so clearly it's missing stuff. Same for the C.txt.

I'm also not sure that it will be super useful as we won't have an "alert" anyway when something is undocumented. I agree with you that it's important not to miss stuff, but maybe  this more easily  achieved by encouraging reviewers to be more careful about this?

Reply:
I'm not really sure on this one, it looks fine from both side, having it is kind of cross-check. Maybe it can save reviewers time? Not too sure again.

Reply:
I don't think we'll ever check this out TBH, we need simple processes with simple signals when reviewing :)

Also, this seems to break the artifacts of CircleCI `build_docs`, so I would suggest to remove